ts/twitter/src/app/posts/[id]/__test__/post-detail.spec.tsx

```diff
@@ -0,0 +1,36 @@
+import { render, screen, fireEvent } from "@testing-library/react";
+import { PostDetail } from "../_components/post-detail";
+
+describe("Post Detail Tests", () => {
+  test("Rendering PostDetail should success", () => {
+    render(<PostDetail />);
+  });
+  test("Follow button display in initial render", () => {
```

Comment:
"display" -> "displays". Also, please add a blank line between each test for better readability.

```suggestion
  test("Rendering PostDetail should success", () => {
    render(<PostDetail />);
  });
  
  test("Follow button displays in initial render", () => {
```